cmake/features.cmake

```diff
@@ -107,7 +107,7 @@ ie_option (ENABLE_IR_V7_READER "Enables IR v7 reader" ${ENABLE_IR_V7_READER_DEFA
 
 ie_option (ENABLE_GAPI_PREPROCESSING "Enables G-API preprocessing" ON)
 
-ie_option (ENABLE_MULTI "Enables Multi Device Plugin" ON)
+ie_option (ENABLE_AUTO "Enables Multi Device Plugin" ON)
```

Comment:
should be ie_option (ENABLE_AUTO "Enables **_AUTO_** Device Plugin" ON) ?

Reply:
Updated.

Reply:
plugin still names as `MULTI`, why the option is renamed?

Reply:
@ilya-lavrenov CCG prefer AUTO as name of this combined plugin for AUTO and MULTI

Reply:
After sync with @myshevts , we keep both the ENABLE_MULTI option and the ENABLE_AUTO option. Auto plugin will be available if any of those two options is enable.